sms_939/models/sms_child_request.py

```diff
@@ -41,3 +43,47 @@ def send_step1_reminder(self):
             })
             sms_sender.send_sms_request()
         super(SmsRequest, self).send_step1_reminder()
+
+    @job(default_channel='root.sms_request')
+    @related_action(action='related_action_sms_request')
+    def reserve_child(self):
+
+        if not self.event_id:
+            return self._take_child_from_website()
```

Comment:
This will add the fact that if no children are found on the website, we call the super to still take it from global childpool.
```suggestion
            res = self._take_child_from_website()
            if res:
                return res
```